### DISCUSSION TITLE
Copy to clipboard

### DISCUSSION THREAD

**Discussion by Sakoche31** [chipzoller/hugo-clarity#195] [General]
Hello,

Thanks to you for your job !!! :)
I don't find option for copy to clipboard with code. 
exemple : 
```powershell
TEST
```

Thanks for your help
Regards

**Comment by chipzoller** [comment]
You'll have to provide more info and show a screenshot or point us to your public site where this isn't happening.

**Comment by Sakoche31** [comment]
Hello,
thanks for your help, this is an example : 

![copy-example](https://user-images.githubusercontent.com/58133862/125251797-fa5b0280-e2f7-11eb-99e4-7c4c67c6d79b.png)

Regards

**Reply by onweru** [reply]
Umh, this doesn't look right. It seems like this is a modified copy of the theme. Unless @Sakoche31 can share their repo, it's futile trying to figure out what's wrong.

**Comment by Sakoche31** [comment]
hum, i clone your repo ... i try to reinstall your theme :)

**Reply by Sakoche31** [reply]
I didn't copy all files in  exemplesite to main folder for understanding. It's ok now !
Thanks a lot
